Author: Sergei Trofimovich <slyfox@gentoo.org>
Date:   Mon Jul 5 22:23:31 2010 +0300

    Fixing link failure og compiler on ia64 ('-Wl,' prefixed value passed directly to ld)
    
    /usr/bin/ld -Wl,--relax -r -o dist-stage1/build/HSghc-6.10.4.o \
                                  dist-stage1/build/BasicTypes.o dist-stage1/build/DataCon.o ...
    /usr/bin/ld: unrecognized option '-Wl,--relax'
    
    Just dropping '-Wl,' part
    
    Signed-off-by: Sergei Trofimovich <slyfox@inbox.ru>

diff --git a/compiler/ghc.mk b/compiler/ghc.mk
index 6b25efa..47b9773 100644
--- a/compiler/ghc.mk
+++ b/compiler/ghc.mk
@@ -340,7 +340,7 @@ endif
 ifeq "$(GhcUnregisterised)" "NO"
 ifeq "$(HOSTPLATFORM)" "ia64-unknown-linux"
 # needed for generating proper relocation in large binaries: trac #856
-compiler_CONFIGURE_OPTS += --ld-option=-Wl,--relax
+compiler_CONFIGURE_OPTS += --ld-option=--relax
 endif
 endif
 
